website/docs/faqs/Environments/delete-environment-job.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to delete a job or environment in dbt Cloud?
+description: "How to delete a job or environment"
+sidebar_label: 'Delete a job or environment'
+id: delete-environment-job
+--- 
+
+
+To delete an environment or job in dbt Cloud, you must have a `developer` [license](/docs/collaborate/manage-access/seats-and-users) and have the necessary [access permissions](/docs/collaborate/manage-access/about-access). 
+
+:::info 📌 Delete a job first before deleting environment
+
+Deleting an environment doesn't automatically delete its associated job(s). If you delete an environment first without deleting the job, you won't be able to delete the job since it's without an environment. 
+
+To completely delete your environment, you _must_:
+1. First delete all jobs associated with that environment,
+2. Then, delete the environment. 
+:::
+
+**Delete a job**
+
+To delete a job or multiple jobs in dbt Cloud:
+
+1. Click **Deploy** on the navigation header.
+2. Click **Jobs** and select the job(s) you want to delete. 
+3. Click **Settings** on the top right of the page and then click **Edit**.
+4. Scroll to the bottom of the page and click **Delete** to delete the job. <br />
+
+<figure>
+<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-job.jpg"/>
+<figcaption align = "center">Delete a job</figcaption>
+</figure>
+
+5. Confirm your action in the **Confirm Delete** pop up by clicking **Confirm Delete** in the bottom right to immediately delete the job. This action cannot be undone. However, you can create a new job with the same information if the deletion was made in error. 
+
+Great job! Refresh your page and the deleted job should now be gone. If you want to delete multiple jobs, you'll need to perform these steps to delete each job. 
+
+**Delete an environment**
+
+To delete an environment in dbt Cloud:
+
```

Comment:
should we have document this for dbt Core as well? if i understand correctly, i think dbt Core supports the development (dev) environment.

Reply:
ok @nghi-ly confirmed with @dave-connors-3 that core isn't in the scope of this since there isn't an object that represents an environment on the cli.  

i have made the change we discussed yesterday regarding related docs replacing the faq src, can you give it a look? thanks!